### PR TITLE
UHF-6142: Created a project listing paragraph and styled it

### PR DESCRIPTION
**Pull request by lonkaanna**
# [UHF-6142](https://helsinkisolutionoffice.atlassian.net/browse/UHF-6142)

## What was done

* Created a project listing paragraph and styled it

## How to install

* Make sure your instance is up and running on correct branch.
  * `git pull origin UHF-6142-project-listing`
  * `make fresh`
  * Update the HDBT theme
      * `composer require drupal/hdbt:dev-UHF-6142-project-listing`
* Run `make drush-cr`

## How to test

* [x] Check that this feature works
* [x] Check that code follows our standards
* [x] Create some dummy projects to your local with the same district (more than 4 so you can test the load more feature )
* [x] Add the new paragraph to a district page (upper content) and select the same district you added to the projects

## Designers review

* [ ] This PR does not need designers review
* [x] This PR has been visually reviewed by a designer (Name of the designer)

## Other PRs

* https://github.com/City-of-Helsinki/drupal-hdbt/pull/376

### CODE REVIEW

File: public/themes/custom/hdbt_subtheme/src/scss/06_components/paragraphs/_project-list.scss [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+.project-list_count-text {
+  @include font('body');
+}
+
+.project-list__count-container {
+  margin-bottom: $spacing-and-half;
+}
+
+.project-list__buttons {
+  list-style: none;
```
Comment (teroelonen) [inline public/themes/custom/hdbt_subtheme/src/scss/06_components/paragraphs/_project-list.scss:22]: Alphabetic order mixed.

File: public/themes/custom/hdbt_subtheme/templates/paragraph/paragraph--project-listing.html.twig [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                </a>
+              </li>
+              {% endif %}
+            </ul>
+          {% endif %}
+        </div>
+
+      {% endif %}
+
+		{% endblock component_content %}
```
Comment (teroelonen) [inline public/themes/custom/hdbt_subtheme/templates/paragraph/paragraph--project-listing.html.twig:49]: Indentation bit funny :)

File: public/themes/custom/hdbt_subtheme/src/js/project-listing.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+(function ($, Drupal, drupalSettings) {
```
Comment (teroelonen) [inline public/themes/custom/hdbt_subtheme/src/js/project-listing.js:3]: This is not needed if changing to view.

**Review by teroelonen — CHANGES_REQUESTED** [review]
Added some comments so that they are not forgotten :)

**Review by tapsal — APPROVED** [review]
Works great! 🦕

**Review by teroelonen — APPROVED** [review]
This is good to go now 🦖

**Pull request merged**